gradle/libs.versions.toml

```diff
@@ -1,34 +1,34 @@
 [versions]
 # Kotlin
-kotlin = "1.8.10"
+kotlin = "1.8.21"
 
 koin = "3.3.3"
 koinCompose = "3.4.2"
 
 # Compose
-activityCompose = "1.6.1"
-compose = "1.3.3"
-composeCompiler = "1.4.3"
+activityCompose = "1.7.1"
+compose = "1.4.2"
```

Comment:
Have you thought about `androidx.compose:compose-bom:2023.05.00`?

Reply:
Done, thanks